### PR TITLE
feat: fixes sitemap and canonical issues

### PR DESCRIPTION
**Pull request by andrewmosesdev**

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  async rewrites() {
```
Comment (brandonburrus) [inline next.config.js]: How does this work when using static site generation?

Comment (andrewmosesdev) [reply next.config.js]: shit, i forgot we're only doing ssg. i can do the sitemap manually through search console, i'll just keep the canonical fix. good call lol

Comment (brandonburrus) [reply next.config.js]: are we able to use something like [next-sitemap](https://www.npmjs.com/package/next-sitemap) to do the generation at build time?

Comment (andrewmosesdev) [reply next.config.js]: i'm checking that out now 👀 could be cool

**Review by brandonburrus — APPROVED** [review]

**Comment by brandonburrus** [comment]
LGTM

**Pull request merged**